Install headers needed by Kopete's cryptography plugin.

Patch borrowed from Fedora.

diff --git a/kleopatra/libkleopatraclient/CMakeLists.txt b/kleopatra/libkleopatraclient/CMakeLists.txt
index 17b4980..690b00d 100644
--- a/kleopatra/libkleopatraclient/CMakeLists.txt
+++ b/kleopatra/libkleopatraclient/CMakeLists.txt
@@ -6,3 +6,7 @@ add_subdirectory(gui)
 
 add_subdirectory(tests)
 
+install(
+  FILES kleopatraclient_export.h
+  DESTINATION ${INCLUDE_INSTALL_DIR}/libkleopatraclient
+  )
diff --git a/kleopatra/libkleopatraclient/core/CMakeLists.txt b/kleopatra/libkleopatraclient/core/CMakeLists.txt
index d4f1ea2..17a6eb5 100644
--- a/kleopatra/libkleopatraclient/core/CMakeLists.txt
+++ b/kleopatra/libkleopatraclient/core/CMakeLists.txt
@@ -63,3 +63,13 @@ else()
 endif()
 
 install(TARGETS kleopatraclientcore ${INSTALL_TARGETS_DEFAULT_ARGS})
+
+install(
+  FILES
+    initialization.h
+    command.h
+    selectcertificatecommand.h
+    signencryptfilescommand.h
+    decryptverifyfilescommand.h
+  DESTINATION ${INCLUDE_INSTALL_DIR}/libkleopatraclient/core
+  )
diff --git a/kleopatra/libkleopatraclient/gui/CMakeLists.txt b/kleopatra/libkleopatraclient/gui/CMakeLists.txt
index 3c5bab2..78dde2e 100644
--- a/kleopatra/libkleopatraclient/gui/CMakeLists.txt
+++ b/kleopatra/libkleopatraclient/gui/CMakeLists.txt
@@ -16,3 +16,9 @@ set_target_properties(kleopatraclientgui PROPERTIES
 )
 
 install(TARGETS kleopatraclientgui ${INSTALL_TARGETS_DEFAULT_ARGS})
+
+install(
+  FILES
+    certificaterequester.h
+  DESTINATION ${INCLUDE_INSTALL_DIR}/libkleopatraclient/gui
+  )
